contents/handbook/strong-team.md

```diff
@@ -0,0 +1,40 @@
+
+**Genuine builders.** Some people do jobs for the money. Those that have truly found their passion are _far_ stronger.
+
+**Easy to work with.** People who are low ego, flexible, energetic, and upbeat will raise those around them. We often, but don't exclusively, hire those with more experience since it's easier for them to contribute meaningfully. Things can and do get _very_ hard here - whether it's scaling, shipping complex products, handling a stream of support requests, or trying to ship something that touches multiple teams. We need those who won't get disheartened, and will collaborate, iterate, and ship their way out of anything. We proactively reward those that do these things, not those that self-promote.
+
+**Will join us on the journey.** Some people are inspirational to work with - they lift others up. We have a _huge_ opportunity at PostHog, and it often feels like we've caught lightning in a bottle. _Anyone joining the company at this stage could make this the last job we all ever need._ We want people that will push to get this done, for each other's sake. We don't hire mercenaries. We need to feel people here are producing the best work of their lives.
+
+**Drivers not passengers.** Proactive people that can fully own projects and get them done (or make sure they get help) are what we need. For many of our roles, while it isn't a common job title, internally we have the concept of [product engineers](/blog/what-is-a-product-engineer) - people who can take high level requirements, decide what to build, do so with customers, and keep iterating.
+
+## Great (and terrible) reasons to join us
+
+Let's start with **why you _should_ join:**
+
+* You want to ship an epic product with incredible people.
+* You want impact and autonomy, and work well with uncertainty.
+
+**Why you _should not_ join:**
+
+* Getting our brand on your resumé. If you join for self-promotion, you (ironically) won't do well! Apply to a bigger company who can give you a clearer career ladder.
+* Getting a pay rise. We pay generously, but you'll need to _love_ building to be happy here. You'll need to be here a long time to get the real upside from options.
+* Mainly wanting to lead others. Reluctant managers are often the best. We don't pay more if you manage others. We want people to lead by example - by doing an exceptional job of individual work.
+
+## A small group of stronger people and compensation
+
+When we raised our Series A, one of the first things we did was to make sure we didn't lose our existing team (at least for pay reasons!) _before_ we added more people to it. Today this is still true - we proactively review everyone's pay 3-4 times a year and increase it if people have leveled up.
+
+When it comes to churn due to pay, fairness is often more important than the absolute level. We do this in line with a transparent pay system that we even [make public](/handbook/people/compensation).
```

Comment:
This first sentence makes it sound like we don't pay well.

Reply:
updated